Tests/Acceptance/TokenEndpointTest.php

```diff
@@ -20,12 +20,22 @@ protected function setUp(): void
     {
         parent::setUp();
 
+        /** @var ScopeManagerInterface $scopeManager */
+        $scopeManager = $this->client->getContainer()->get(ScopeManagerInterface::class);
```

Comment:
All changes here are unrelated to Symfony 5.

Reply:
fixed